src/main/java/net/minecraftforge/fml/packs/ModFileResourcePack.java

```diff
@@ -27,6 +27,7 @@
 import net.minecraftforge.fml.loading.moddiscovery.ModFile;
 
 import java.io.File;
+import java.io.FileNotFoundException;
```

Comment:
This import isn't needed anymore since the used exception is `ResourcePackFileNotFoundException` again, right?